src/modules/AutoinstStorage.rb

```diff
@@ -359,6 +387,26 @@ def build_id(disk_dev_name, nr)
     publish :function => :ImportAdvanced, :type => "boolean (map)"
     publish :function => :Summary, :type => "string ()"
     publish :function => :Write, :type => "boolean ()"
+
+  private
+
+    attr_accessor :general_settings
+
+    # Initialize partition proposal
+    #
+    # @return [Boolean] true if proposal was successfully created; false otherwise.
+    def initialize_proposal
+      log.info "Initializing a proposal"
+      proposal_settings = Y2Storage::ProposalSettings.new_for_current_product
+      proposal = Y2Storage::Proposal.new(settings: proposal_settings)
```

Comment:
It was already [there](https://github.com/yast/yast-autoinstallation/blob/ec304d3a1b91e027510eb3826dfacee0537ec8d2/src/modules/AutoinstStorage.rb#L10).